stripe-buffer.el

```diff
@@ -136,9 +174,25 @@ Used by `stripe-table-mode' Only the first matching group will be painted."
                                     (goto-char end)
                                     (point))))))
                       ( overlay (apply get-overlay-create-function stripe-region)))
-                 (overlay-put overlay 'face stripe-highlight-face)
-                 (overlay-put overlay 'is-stripe t)
-                 (push overlay sb/overlays)))))
+
+                 (cond
+                  ((unless (require 'dired-subtree nil t)
```

Comment:
Thanks.
I fixed it again. https://github.com/myuhe/stripe-buffer/commit/3a980dd7491c98fc96d25caa181f31545ac6d181
check it.